frontend/server/src/DAO/Problems.php

```diff
@@ -1147,12 +1150,19 @@ public static function hasBeenUsedInCoursesOrContests(
                 ON
                     t.tag_id = pt.tag_id
                 WHERE
-                    t.name = ?;';
+                    t.name LIKE CONCAT('problemLevel','%')
+                GROUP BY
+                    t.name;";
 
-         /** @var int */
-        return \OmegaUp\MySQLConnection::getInstance()->GetOne(
-            $sql,
-            [$problemLevelTag]
-        );
+        $result = [];
+        /** @var array{alias: string, problems_per_tag: int} $row */
+        foreach (
+            \OmegaUp\MySQLConnection::getInstance()->GetAll(
+                $sql
+            ) as $row
+        ) {
+            $result[] = new \OmegaUp\DAO\VO\Problems($row);
+        }
+        return $result;
```

Comment:
Como `GetAll` ya regresa una lista de todo lo que se obtiene en la consulta, y no es necesario hacer una instancia de `\OmegaUp\DAO\VO\Problems` porque `problems_per_tag` no es un atributo de ella, puedes simplificarlo de esta forma:
```suggestion
        /** @var array{alias: string, problems_per_tag: int} */
        return \OmegaUp\MySQLConnection::getInstance()->GetAll($sql);
```
?